.github/workflows/release.yaml

```diff
@@ -96,3 +96,21 @@ jobs:
           generateReleaseNotes: true
           makeLatest: true
           body: ${{ steps.Changelog.outputs.changelog }}
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.EMA_AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.EMA_AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ secrets.EMA_AWS_DEFAULT_REGION }}
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1.6.0
+      - name: ECR Docker Image Release
+        run: |
+          ECR_REPO="${{ steps.login-ecr.outputs.registry }}/${{ github.event.repository.name }}"
+          MAIN_IMAGE="$ECR_REPO:main"
+          RELEASE_IMAGE="$ECR_REPO:${{ github.event.inputs.releaseVersion }}"
+          echo "Release image : $RELEASE_IMAGE" 
+          docker pull $MAIN_IMAGE  
```

Comment:
It's a one off task during release, but typically if it's coming from ECR it's better to use ECR's update tag thru the CLI than pulling the image and retagging it. Pulls depending on how big the image is, can be expensive

Reply:
This is a snippet we use for Jenkins when we tag an image, we don't pull it, but rather get the manifest metadata and then add an image tag based on the metadata.
```
MANIFEST = $("aws ecr batch-get-image --repository-name ${repository} --image-ids imageTag=${image_Tag} --query 'images[].imageManifest' --region us-east-1 --output text")
 aws ecr put-image --repository-name ${repository} --image-tag ${tag} --image-manifest \"\$MANIFEST\" --region us-east-1
```

Reply:
That's a great idea, going to use this approach.